server/helpers/sms.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const { KUDI_SMS_USERNAME, KUDI_SMS_PASSWORD, KUDI_SMS_SENDERNAME } = process.env;
+
+const KUDI_API_URL = `https://account.kudisms.net/api/?username=${KUDI_SMS_USERNAME}&password=${KUDI_SMS_PASSWORD}&`;
+
+export const sendSms = async ({ number, message }) => {
```

Comment:
Lets use phoneNumber to make it easily understandable.

```suggestion
export const sendSms = async ({ phoneNumber, message }) => {
```